index.js

```diff
@@ -66,6 +66,10 @@ module.exports = (dest, opts) => {
 		throw new gutil.PluginError('gulp-changed', '`dest` required');
 	}
 
+	if (opts.transformPath !== undefined && typeof opts.transformPath !== 'function') {
```

Comment:
If I remove the first check, and the user does not supply the `transformPath` property, `typeof opts.transformPath` will result in `"undefined"`. And thus throw the error on line 70.